js/src/logger.ts

```diff
@@ -1960,6 +1973,13 @@ export function startSpan<IsAsyncFlush extends boolean = false>(
   return startSpanAndIsLogger(args).span;
 }
 
+/**
+ * Flush any pending rows to the server.
+ */
+export async function flush(): Promise<void> {
```

Comment:
I added a global flush method in case people want to flush records not attached to any particular toplevel object, such as records produced from distributed tracing. Now all of the flush functions flush the same logger, so technically this is the only flush function anybody should need.